mmaction/datasets/pipelines/augmentations.py

```diff
@@ -195,31 +189,42 @@ def __call__(self, results):
 class EntityBoxCrop:
     """Crop the entity boxes and proposals according to the cropped images.
 
-    Required keys are "proposals", "ann.entity_boxes", "crop_bbox", added or
-    modified keys are "ann.entity_boxes". If original "proposals" is not None,
-    "proposals" will be modified.
+    Required keys are "proposals", "gt_bboxes", added or modified keys are
+    "gt_bboxes". If original "proposals" is not None, "proposals" will be
+    modified.
+
+    Args:
+        crop_bbox(np.ndarray | None): The bbox used to crop the original image.
     """
 
+    def __init__(self, crop_bbox):
+        self.crop_bbox = crop_bbox
```

Comment:
Are `crop_bbox` for all data the same?

Reply:
Same as above, the crop_bbox is designated by XXCrop ops.